Core/ClassConfig/ClassConfiguration.cs

```diff
@@ -27,6 +27,7 @@ public class ClassConfiguration
         public bool Loot { get; set; } = true;
         public bool Skin { get; set; } = false;
         public bool UseMount { get; set; } = true;
+        public bool Background { get; set; } = false;
```

Comment:
The variable name is quite misleading and at first it was hard to wrap around my head. Maybe it would be better to find a more suitable name for this "Mode".

Reply:
Like `KeyboardOnlyMode`

Reply:
Instead of creating a new boolean would recommend to check out the 

https://github.com/Xian55/WowClassicGrindBot/blob/1d4802701fe20832829f439245c5a2462de4209d/Core/ClassConfig/ClassConfiguration.cs#L39

Reply:
Yes indeed it could use a better name. But it comes to me that it actually does not have to work with Grind mode only.